docs/topics/faq.md

```diff
@@ -128,10 +128,14 @@ on the [Books](books.md) page. For more books, see the community-maintained list
 
 ### Are there any online courses available for Kotlin?
 
-There are a few courses available for Kotlin, including a [Pluralsight Kotlin Course](https://www.pluralsight.com/courses/kotlin-getting-started) by Kevin Jones, 
-an [O’Reilly Course](https://shop.oreilly.com/product/0636920052982.do) by Hadi Hariri and an [Udemy Kotlin Course](https://petersommerhoff.com/dev/kotlin/kotlin-beginner-tutorial/) by Peter Sommerhoff.
+You can learn all the Kotlin essentials while creating working applications with the [Kotlin Basics track](https://hyperskill.org/tracks/18) on JetBrains Academy.
```

Comment:
Has this sentence been proofread? 🤔
TBH, I don't like "while creating working" here

Reply:
Yeap, checked this line (and other content) with proofreaders